x/superfluid/keeper/twap_price.go

```diff
@@ -33,15 +36,23 @@ func (k Keeper) SetOsmoEquivalentMultiplier(ctx sdk.Context, epoch int64, denom
 	prefixStore.Set([]byte(denom), bz)
 }
 
-func (k Keeper) GetSuperfluidOSMOTokens(ctx sdk.Context, denom string, amount sdk.Int) sdk.Int {
+func (k Keeper) GetSuperfluidOSMOTokens(ctx sdk.Context, denom string, amount sdk.Int) (sdk.Int, error) {
+	// If the denom is a concentrated liquidity token, we just strip the position data from the denom
+	if strings.HasPrefix(denom, cltypes.ClTokenPrefix) {
+		index := strings.LastIndex(denom, "/")
+		denom = denom[:index+1]
```

Comment:
I suspect this might be including "/" in denom. Is that intended?

Reply:
It was, and was intended, but this no longer exists in next PR and follows the same format as gamm shares